dolphinscheduler-python/pom.xml

```diff
@@ -153,14 +153,48 @@
                                     <goal>exec</goal>
                                 </goals>
                                 <configuration>
-                                    <executable>python3</executable>
+                                    <executable>python</executable>
                                     <workingDirectory>${project.basedir}/pydolphinscheduler</workingDirectory>
                                     <arguments>
                                         <argument>-m</argument>
                                         <argument>build</argument>
                                     </arguments>
                                 </configuration>
                             </execution>
+                            <execution>
+                                <id>sign-source</id>
+                                <phase>prepare-package</phase>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <skip>${python.sign.skip}</skip>
```

Comment:
What about setting this property to true in profile `release` so that the gpg signature of Python tar is also generated along with the java's?

Reply:
SGTM, but in this case, the release manager should run `./mvnw clean package -Prelease,python` trigger profile `python`

Reply:
BTW, you mean set `<python.sign.skip>false</python.sign.skip>` to https://github.com/apache/dolphinscheduler/blob/d7e676ded2c3c296f79eebf97618bc33bf1a3f77/dolphinscheduler-dist/pom.xml#L29-L31